api/HTMLInputElement.json

```diff
@@ -1467,7 +1467,7 @@
               "version_added": true
             },
             "edge": {
-              "version_added": true
+              "version_added": "13"
```

Comment:
`webkitdirectory` confirmed missing in Edge 12. It's just a reflected IDL attribute so the content attribute might still work though.